### PR TITLE
Swatch CSS Variables - RSC-549

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-549

In contrast to #383, this PR updates the color swatch variables. It will be merged to a feature branch with the other PR before being released

### CODE REVIEW

File: gallery/tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "ES5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "target": "ES2018",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```
Comment (rpokorny) [inline gallery/tsconfig.json:4]: This change is necessary to get the code that extracts the swatch names in `ColorPaletteExample` to work

File: gallery/visualtests/nxToggle.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ describe('NxToggle', function() {
 
       await targetElement.scrollIntoView({ block: 'center' });
       await targetElement.click();
+      await targetElement.moveTo({ xOffset: -10, yOffset: -10 });
```
Comment (rpokorny) [inline gallery/visualtests/nxToggle.js:28]: Fixes issue @dturner32 noticed where hover style was present in this applitools check

**Comment by glenhunter** [comment]
I don't know if we care that much since it's just the gallery but at `1366px` some of the content overflows the swatch cards.

**Comment by rpokorny** [comment]
> I don't know if we care that much since it's just the gallery but at 1366px some of the content overflows the swatch cards.

4c6e0220

**Comment by glenhunter** [comment]
`_nx-tag.scss` still has a couple of hex colour values in it.

**Comment by rpokorny** [comment]
> I'm also curious why we've kept the fallback values in NxTag.scss

You mean the second arguments in the `var()` calls? That's how the "default" NxTag is implemented

**Comment by rpokorny** [comment]
> NxTag.scss still has a couple of hex colour values in it.

Using swatch colors per @dturner32 : 82503c79

**Review by glenhunter — APPROVED** [review]

**Pull request merged**